okta/error_messages.py

```diff
@@ -13,7 +13,7 @@
 ERROR_MESSAGE_AUTH_MODE_INVALID = (
     "The AuthorizationMode configuration "
     "option must be one of: "
-    "[SSWS, PrivateKey]. "
+    "[SSWS, Bearer, PrivateKey]. "
```

Comment:
What happens if devs set "PrivateKey"?

Reply:
It will get access token automatically and pass it within Bearer auth mode under the hood: https://github.com/okta/okta-sdk-python/blob/master/okta/request_executor.py#L127-L141